src/index.css

```diff
@@ -6,6 +6,10 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+* {
+  margin: 0;
+}
```

Comment:
:open_mouth: What margins being removed with this? Would this override many of the MUI pre-defined margins?

Reply:
no, mui has its own def, I think we should just have a reset.css somewhere tbh, but I opted in for this only because of how the designer expect the header to work vs how the header margin actually work..